### PR TITLE
Rearchitect and update docs

### PR DESCRIPTION
**Pull request by smoreface**
Updating content to be more quickstart/overview oriented.

After reading this page I can explain to my colleague which parts of Label Studio I might want to extend
After reading this page I can map out the steps of a data labeling project that I need to prepare for.

### CODE REVIEW

File: docs/source/guide/FAQ.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-title: FAQ
+title: Troubleshoot Label Studio
```
Comment (makseq) [inline docs/source/guide/FAQ.md:2]: @smoreface Why FAQ is a not good variant?

Comment (smoreface) [reply docs/source/guide/FAQ.md:2]: Most of the time FAQs aren't actually helpful to readers -- see [this article about why](https://content-guide.18f.gov/our-approach/structure-the-content/#dont-use-faqs). Most importantly for us, most of the content on this page isn't actually a question, even if it is frequently asked.

I plan to change this page into a troubleshooting page for errors (like CORS etc.) and then move the other helpful content (like the setting up pre-labeling) to dedicated pages or more relevant pages (like a second machine learning page about setting up and working with pre-labels, in addition to the existing page about setting up machine learning backends!)

**Comment by smoreface** [comment]
Planning to merge this on Monday and work on 1.0.0 docs in a different branch!

**Comment by atakanokan** [comment]
Would it be possible to address this issue before the merge too? 

https://github.com/heartexlabs/label-studio/issues/625

**Comment by makseq** [comment]
@smoreface I think you should connect with @hlomzik (Andrey) to clarify the question about selectedColor.

**Comment by smoreface** [comment]
@atakanokan I'll address that issue in a future PR, this one doesn't address the tags documentation at all.

**Review by niklub — APPROVED** [review]

**Pull request merged**